packages/gatsby-plugin-cxs/gatsby-ssr.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");
+
+var _react = _interopRequireDefault(require("react"));
+
+var _cxs = _interopRequireDefault(require("cxs"));
+
+var _jsxFileName = "/Users/ctxhou/Program/Github/gatsby/packages/gatsby-plugin-cxs/src/gatsby-ssr.js";
```

Comment:
Whoa! So looks like the transpiled copy made it's way up. We'll want to add a gitignore to make sure we resolve this!

Reply:
😅